examples/use-cases/vuln_sync_csv_upload/vuln_upload.py

```diff
@@ -0,0 +1,197 @@
+"""Use case for automating the ingestion of CVE reports"""
+import os
+import sys
+import time
+from getpass import getpass
+from io import FileIO
+
+import requests
+
+
+def upload_cve(file: FileIO):
+    """ Uploads vulnerability list to Automox Vulnerability Sync endpoint.
+
+    Args:
+        file (FileIO): A CSV file containing vulnerability data.
+
+    Returns:
+        response_data (dict): API response from Automox Vulnerability Sync
+        https://developer.automox.com/openapi/vulnsync/operation/UploadCSVBatch/
+    """
+
+    response_data = {}
+
+    task = "patch"
+    url  = f"https://console.automox.com/api/orgs/{organization}/tasks/{task}/batches/upload"
```

Comment:
Wasn't there something where the API endpoints needed you to also specify the `o=<org_id>` query parameter or was that changed?

Reply:
I don't think so.

The docs only reference path params: https://developer.automox.com/openapi/vulnsync/operation/UploadCSVBatch/

Reply:
👍